package.json

```diff
@@ -39,6 +39,7 @@
     "ethereumjs-util": "5.2.0",
     "ethereumjs-vm": "2.6.0",
     "heap": "0.2.6",
+    "js-scrypt": "^0.2.0",
```

Comment:
This needs to be moved to `devDependencies` and pinned to `0.2.0`. The shrinkwrap file will have to be updated once moved as well.

Reply:
Agreed, it should be in devDependencies. Well spotted! I'll push an updated package.json in a minute.

Just out of curiosity, why would you like to have it pinned to 0.2.0? Shouldn't the npm-shrinkwrap.json take care of that?

Reply:
Thanks.

For `yarn` users and so we can better control when we update transitive dependencies. If a developer working on ganache-core runs `npm install` it will automatically _update_ the shrinkwrap file, which can make things difficult to maintain and test consistently if dependencies are always updating (which still happens for transitive dependencies, unforatunely). Running `npm ci` avoids this, but isn't what it is meant for a takes longer to run (as it wipes out the node_modules dir first).

So, we try to keep core dependencies (dependencies used by core, and not just by our test suite) pinned.

Reply:
Thanks, that makes sense.